pom.xml

```diff
@@ -90,6 +90,10 @@
     <node.version>you-must-override-the-node.version-property</node.version>
     <npm.version>you-must-override-the-npm.version-property</npm.version>
     <yarn.version>you-must-override-the-yarn.version-property</yarn.version>
+    <!-- can be set to the empty string or install for yarn2 -->
+    <!-- ensure only one concurrent 'yarn install' -->
+    <!-- when yarn cache is empty, multiple yarns performing network fetches frequently results in opaque errors -->
+    <yarn.args>---mutex network</yarn.args>
```

Comment:
https://github.com/yarnpkg/yarn/issues/2146#issuecomment-292609718 you can as well create a `.yarnrc` and add `--install.mutex network` in case you want it.